app/views/notification_mailer/reject_email.html.erb

```diff
@@ -0,0 +1,12 @@
+<p>Dear <%= @user.name %>,</p>
+
+<p>Your deposit, “<%= @work.title %>” to the <%= @work.collection_name %>
+  collection in the Stanford Digital Repository, has been returned to you with the following  comments:</p>
```

Comment:
Extra space.

Reply:
Fixed, but this is exactly what was asked for in https://docs.google.com/document/d/1TZVILUrORI25kRZyUZ70vMQYo90Ngp_tqCI7BdU_VyQ/edit#heading=h.pyrjq486puns